lib/src/androidMain/kotlin/dev/zt64/compose/pdf/RemotePdfState.android.kt

```diff
@@ -0,0 +1,128 @@
+package dev.zt64.compose.pdf
+
+import android.graphics.pdf.PdfRenderer
+import android.net.Uri
+import android.os.ParcelFileDescriptor
+import androidx.compose.runtime.*
+import androidx.compose.runtime.saveable.Saver
+import androidx.compose.runtime.saveable.listSaver
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.graphics.vector.rememberVectorPainter
+import dev.zt64.compose.pdf.util.downloadPdf
+import dev.zt64.compose.pdf.util.renderPage
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.cancel
+import java.net.URL
+
+@Stable
+public actual class RemotePdfState actual constructor(
+    private val url: URL,
+    public var errorPainter: Painter,
+    public var loadingPainter: Painter
+) : PdfState {
+    private val coroutineScope = CoroutineScope(Dispatchers.IO)
+
+    private var pfd: ParcelFileDescriptor? = null
+    private var renderer: PdfRenderer? = null
+
+    actual override var pageCount: Int by mutableIntStateOf(1)
+    public actual var loadState: LoadState by mutableStateOf(LoadState.Loading)
```

Comment:
you forgor to make setter private, or internal
also pageCount should be marked public i think

Reply:
weird that my ide wasn't yelling at me about marking that public